mobileapp/src/components/screens/notification/NotificationDetailScreen/NotificationDetailScreen.tsx

```diff
@@ -92,13 +118,121 @@ export const NotificationDetailScreen: React.FC<
     setErrorDialog(false)
   }
 
+  const DetailsNotificationId = () => {
+    return (
+      <Box alignItems="center">
+        <Text fontSize={20} fontWeight="bold">
+          {R.strings.details.id}
+        </Text>
+        <Text italic fontSize={17}>
+          {details?.id}
+        </Text>
+      </Box>
+    )
+  }
+
+  const DetailsNotificationType = () => {
+    return (
+      <Box alignItems="center">
+        <Text fontSize={20} fontWeight="bold">
+          {R.strings.details.type}
+        </Text>
+        <Text italic fontSize={17}>
+          {details?.type}
+        </Text>
+      </Box>
+    )
+  }
+
+  const DetailsNotificationMessage = () => {
+    return (
+      <Box alignItems="center">
+        <Text fontSize={20} fontWeight="bold">
+          {R.strings.details.message}
+        </Text>
+        <Text italic fontSize={17}>
+          {details?.message}
+        </Text>
+      </Box>
+    )
+  }
+
+  const DetailsDestinationAddress = () => {
+    return (
+      <Box alignItems="center">
+        <Text fontSize={20} fontWeight="bold">
+          {R.strings.details.address}
+        </Text>
+        <Text italic fontSize={17}>
+          {details?.address}
+        </Text>
+      </Box>
+    )
+  }
+
+  const DetailsTransactionAmount = () => {
+    return (
+      <Box alignItems="center">
+        <Text fontSize={20} fontWeight="bold">
+          {R.strings.details.amount}
+        </Text>
+        <Text italic fontSize={17}>
+          {details?.amount}
+        </Text>
+      </Box>
+    )
+  }
+
+  const DetailsNotificationStatus = () => {
+    return (
+      <Box alignItems="center">
+        <Text fontSize={20} fontWeight="bold">
+          {R.strings.details.status}
+        </Text>
+        <Text italic fontSize={17}>
+          {details?.status}
+        </Text>
+      </Box>
+    )
+  }
+
+  const DetailsScreenButtons = () => {
+    return (
+      <HStack space={10}>
+        <Button size="lg" w="40%" bg="error.600" variant="solid" onPress={deny}>
+          {R.strings.details.btnDeny}
+        </Button>
+        <Button
+          size="lg"
+          w="40%"
+          bg="success.600"
+          variant="solid"
+          onPress={confirm}
+        >
+          {R.strings.details.btnConfirm}
+        </Button>
+      </HStack>
+    )
+  }
+
+  /**
+   * Return View with data from NotificationDetails
+   */
   return (
-    <Center justifyContent="center" h="100%">
+    <Center justifyContent="flex-start" h="100%">
       <LoadingModal processing={processing} />
       <ErrorAlert isOpen={errorDialog} close={alertClose} />
-      <Text>{notificationId}</Text>
-      <Button onPress={confirm}>confirm</Button>
-      <Button onPress={deny}>deny</Button>
+      <VStack space={3} marginTop={5}>
+        <DetailsNotificationId />
```

Comment:
To powinien być jeden komponent, taki w osobnym pliku. Mamy 6 dokładnie takich samych komponentów, które różnią się tylko danymi. Co jak ci teraz każę zmienić rozmiar czcionki w we wszystkich?.

Reply:
poprawiłem